GermanFilter/sections/specific.txt

```diff
@@ -96,6 +96,7 @@ drwindows.de,wiesbadenaktuell.de,rauchmeldungen.de,simfans.de,wn.de,cosmopolitan
 ! ##.id-Container--proBEEP
 hallo-muenchen.de,az-online.de,lokalo24.de,rosenheim24.de,kreiszeitung.de,nordbuzz.de,mannheim24.de,merkur.de,oktoberfest-live.de,tz.de##.id-Container--proBEEP
 !
+tz.de##.id-RatioPlaceholder.wv_story_el_insertedAd
```

Comment:
`tz.de##.wv_story_el_insertedAd` is enough, and please move it to the mobile filter > specific

Reply:
ok